src/utils/database.ts

```diff
@@ -0,0 +1,37 @@
+const { existsSync, readFileSync, writeFileSync, unlinkSync } = require('fs');
+
+const defaultFilePath = 'saves/data.json';
+
+module.exports = {
+  hasGameFile(): boolean {
+    if (existsSync(defaultFilePath)) return true;
+    return false;
```

Comment:
You can just return `existsSync(defaultFilePath)`

Reply:
done